index.js

```diff
@@ -35,10 +36,12 @@ export default function (app, router, store, config) {
 
       // Dynamically inject a component into the order review section (optional)
       const Component = Vue.extend(PaypalComponent)
-      const componentInstance = (new Component({
-        parent: app
-      }))
+      const componentInstance = (new Component({ parent: app }))
       componentInstance.$mount('#checkout-order-review-additional')
+
+      const Button = Vue.extend(PaypalButton)
+      const btnInstance = (new Button({ parent: app }))
+      btnInstance.$mount('#order-review-container')
```

Comment:
@collymore need add the id with `order-review-container` to this line https://github.com/DivanteLtd/vue-storefront/blob/2cf12749378aebb171d2765b5af0054b8381cba5/src/themes/default/components/core/blocks/Checkout/OrderReview.vue#L65